src/controller/python/ChipDeviceController-ScriptDevicePairingDelegate.cpp

```diff
@@ -19,12 +19,17 @@
 
 #include "ChipDeviceController-ScriptDevicePairingDelegate.h"
 #include "lib/support/TypeTraits.h"
+#include <app/icd/client/DefaultICDClientStorage.h>
 #include <controller/python/chip/native/PyChipError.h>
 #include <setup_payload/ManualSetupPayloadGenerator.h>
 #include <setup_payload/QRCodeSetupPayloadGenerator.h>
 
 #include <string>
 
+extern chip::app::DefaultICDClientStorage sICDClientStorage;
+extern chip::Controller::CommissioningParameters sCommissioningParameters;
+extern uint8_t sICDSymmetricKey[chip::Crypto::kAES_CCM128_Key_Length];
```

Comment:
Direct global access should not be done. There should be a struct with the necessary state and a way to get a pointer to it. This goes across the wall. It can't be called `sXxx` if it's not static. It would have needed to be `gXxx`

Reply:
https://github.com/project-chip/connectedhomeip/issues/33733